omas/machine_mappings/d3d.py

```diff
@@ -523,15 +523,16 @@ def interferometer_hardware(ods, pulse):
     ods['interferometer.channel.0.identifier'] = ods['interferometer.channel.0.name'] = 'r0'
     r0 = ods['interferometer.channel.0.line_of_sight']
     r0['first_point.phi'] = r0['second_point.phi'] = 225 * (-np.pi / 180.0)
-    r0['first_point.r'], r0['second_point.r'] = 3.0, 0.8  # These are not the real endpoints
```

Comment:
@AreWeDreaming you may want to remove the comment above: "The chord endpoints ARE NOT RIGHT. Only the R for vertical lines or Z for horizontal lines is right.".